src/framework/components/element/constants.js

```diff
@@ -18,3 +18,24 @@ export const ELEMENTTYPE_IMAGE = 'image';
  * @type {string}
  */
 export const ELEMENTTYPE_TEXT = 'text';
+
+/**
+ * Disable Aspect Ratio content resizing.
+ *
+ * @type {number}
+ */
+export const ELEMENT_IMAGE_ASPECT_NONE = 0;
```

Comment:
I wonder if those should be strings instead of numbers, for easier way to debug things.

Reply:
I was a bit worried if there'd be any noticeable performance implications in UI-heavy scenes, considering it needs to be checked every time the mesh changes.

An alternative perhaps is to have a "public" property which is a string, but then gets converted to a number on `set`.

Reply:
There would be no difference, as constant strings are compared by pointers, so the speed is the same as numbers.

Reply:
Oh nice - didn't know that! I'll change it to strings, then.